pennylane_lightning/lightning_gpu/_state_vector.py

```diff
@@ -17,7 +17,7 @@
 from warnings import warn
 
 try:
-    from pennylane_lightning.lightning_gpu_ops import StateVectorC64, StateVectorC128
+    from pennylane_lightning.lightning_gpu_nb import StateVectorC64, StateVectorC128
 
     try:  # Try to import the MPI modules
         from pennylane_lightning.lightning_gpu_ops import StateVectorMPIC64, StateVectorMPIC128
```

Comment:
I found some problems with this approach while developing for LQubit. Could you please remove it and make it only look for nanobind binaries? As before, but with the new binaries.